src/main/java/serhat/workshop_5/CatalogPage.java

```diff
@@ -0,0 +1,47 @@
+package serhat.workshop_5;
+
+/**
+ * Created by serhatgemici on 05.04.17.
+ */
+public class CatalogPage extends BasePage {
+
+
+    protected CatalogPage(String partialURL) {
+        super(partialURL);
+    }
+
+    @Override
+    protected void openPage() {
+
+        if (!partialURL.equals("/cars") && !partialURL.equals("/motorcycles") && !partialURL.equals("/commercial-vehicles") && !partialURL.equals("/all")) {
+            System.out.println("Opening URL: " + BASE_URL + partialURL);
+            System.out.println("Error: Cannot handle given partial url /sellcar, page will not be shown");
+        }
+
+        if (partialURL == "/cars") {
```

Comment:
Please, use equals()
Example: http://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java